<?xml version="1.0"?>
<launch>
<!--JOYSTICK-->
	<node respawn="true" pkg="joy" type="joy_node" name="joy_node" >
		<param name="dev" type="string" value="/dev/input/js0" />
		<param name="deadzone" value="0.12" />
		<remap from="joy" to="roboteq_driver/joy"/>
	</node>

<!--CLEARPATH ARDUINO {Brake Feedback and Servo Commands}-->
	<node pkg="rosserial_python" type="serial_node.py" name="ClearPath_serial_node">
		<param name="port" value="/dev/sensors/servo_arduino"/>
	</node>
 
<!--ENCODER ARDUINO-->
   <node name="arduino" pkg="ros_arduino_python" type="arduino_node.py" output="screen">
      <rosparam file="$(find ros_arduino_python)/config/my_arduino_params.yaml" command="load" />
   </node>

<!--ROBOTEQ Node-->
    <node pkg="roboteq_driver" type="driver_node" name="roboteq_driver" output="screen">
		<param name="port" value="/dev/sensors/roboteq" />
		<param name="value" value="9600" />
	</node>

<!-- Servo Commands Node -->
		<param name="axis_linear" value="1" type="int"/>
		<param name="axis_angular" value="0" type="int"/>
		<param name="scale_linear" value="150" type="double"/>
		<param name="scale_angular" value="250" type="double"/>
	<node pkg="argo_base_control" type="argo_base_control_node" name="argo_base_controller"/>


</launch>
<!-- Probably have to split into two namespaces -->
